Example/Models/UserProfileModel.swift

```diff
@@ -30,9 +30,11 @@ struct AccessTokenInfo: Codable {
 struct QueryParamInfo: Codable {
     var queryString: String
 
+    // swiftlint:disable unneeded_synthesized_initializer
     init(queryString: String) {
         self.queryString = queryString
     }
+    // swiftlint:enable unneeded_synthesized_initializer
```

Comment:
Sorry, I did not see this and I have added this rule to SwiftLint.yaml file in my PR#504. Please let me know before merging my PR if I need to revert it.